xml/security_polkit.xml

```diff
@@ -364,7 +380,7 @@ org.freedesktop.login1.reboot:
    <para>
     The file
     <filename>/usr/share/polkit-1/actions/org.opensuse.policykit.gparted.policy</filename>
-    has the following content:
+    contains the following content:
```

Comment:
`contains the following content` sounds repetitive / redundant to me, but maybe it's just me ...

Reply:
It does sound repetitive. Reverted.